php/PhpJsonApiSamples/ClientSideEncryptionDemo1.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Client side Encryption</title>
+        
+        <!-- This page shows Demo for using PayTrace client side Encryption library - after the client side validation are met.-->
+        
+        <!-- This is the PayTrace End-to-End Encryption library: -->
+        <script language="javascript" src="https://api.paytrace.com/assets/e2ee/paytrace-e2ee.js"></script>
+        <!-- Note: PayTrace uses jquery 1.11.1 so please use the same version if possible.Using different versions of jquery can cause some issues.
```

Comment:
this comment where it says if possible, what if it isn't possible. Not sure what we usually do if they require a different Jquery version. I don't know the answer to this. Maybe you don't need to add the "if possible" verbiage.